### DISCUSSION TITLE
So where are the PNG Info and "Do not add watermark to images"?

### DISCUSSION THREAD

**Discussion by poko3347** [vladmandic/sdnext#485] [Q&A]
title

At any rate, the speed of this forked version is excellent.

**Comment by poko3347** [comment]
I understand that png info has been merged into process image.
But I can't seem to pass the info to the t2i tab.

**Reply by vladmandic** [reply]
button is right below the image as soon as you load it

**Comment by vladmandic** [comment]
> Do not add watermark to images

that was always just a dummy option, never actually implemented. removed.
